data-integration/main.bal

```diff
@@ -0,0 +1,103 @@
+import ballerina/http;
+import ballerina/os;
+import ballerina/persist;
+import ballerina/lang.runtime;
+import data_integration.db;
+
+configurable int port = 9090;
+
+service /employees on new http:Listener(port) {
+    private final db:Client dbClient;
+
+    function init() returns error? {
+        _ = check startDBService();
+        self.dbClient  = check new();
+    }
+
+    isolated resource function post employees(db:EmployeeInsert employee)
```

Comment:
Is this API path definition is correct? If someone wants to delete an employee they should send a request like,
`BASE_PATH/employees/employees/.
This is not adhering to the REST_API designing best practices.

Reply:
Added a suitable name for the service